.github/workflows/example.yml

```
@@ -17,13 +17,13 @@ concurrency:
 
 jobs:
   example:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
 
     env:
       ARM_UBL_ACTIVATION_CODE: ${{ secrets.ARM_UBL_ACTIVATION_CODE }}
 
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v4.2.2
```

Let's stick to the version for now. We might want to harden at some point and pin to commit SHA's
```suggestion
      - uses: actions/checkout@v4
```